### PR TITLE
Add travis-ci issue/solution to README

### PR DESCRIPTION
**Pull request by mtrivera**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,3 +47,17 @@ I’m comfortable with git, the Linux environment, ES6, webpack, SASS. This past
...
+
+Issue: Travis CI build failure
+
+Solution (by Miguel T Rivera):
+
+1. Analyzed the build log and noticed it could not recognize ES6 syntax.
+2. Decided it was probably old and that files needed to be updated.
+3. Noticed some files were updated recently and others were 2 years old.
+4. Tried updating the files using git but ran into issues.
+5. Manually replaced each old file with the recent version.
```
Comment (pkshreeman) [inline README.md:61]: Can you please comment on what files you replaced.  That was just what I was looking for. Just simple 'log' information.

Comment (pkshreeman) [inline README.md:61]: Also, Can you please pull request on Draft? I would like to be able to clone it on draft in general before we merge it into master.  You can create your branch here if you want to and makes you feel more comfortable, and just push that into repo so we can see what changes we are doing, and merge it?

**Review by pkshreeman — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mtrivera) [reply README.md:61]: I'll probably close this PR and submit a new one. Also, can put in the readme what the different branches purposes? I wasn't sure which one to submit the PR.

**Comment by pkshreeman** [comment]
@mtrivera that’s my fault.  I recommend that we do all the changes to draft branch.  As for other branches, I am just experimenting.  You can create your own branch, run wild, and if it goes good, pull request for draft.  The master is the “final” deployed copy to run on glitch.

**Pull request closed without merging**